dll/win32/comctl32/listbox.c

```diff
@@ -1945,7 +1951,11 @@ static LRESULT LISTBOX_HandleHScroll( LB_DESCR *descr, WORD scrollReq, WORD pos
         case SB_THUMBTRACK:
             info.cbSize = sizeof(info);
             info.fMask  = SIF_TRACKPOS;
+#ifdef __REACTOS__
```

Comment:
Yes, this is actually correct :) This is the same fix already present in the user32 version of the listbox.